index.html

```diff
@@ -0,0 +1,141 @@
+        </div>
+
+    </section>
+    <!--About section-->
+    <section class="about">
+        <div class="about-content">
+            <h2>About <br> Myself</h2>
+            <p class="my-intro">Hello I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! If you like what
+                 you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <br>
+            <hx class="lets-connect-message">LET'S CONNECT</hx>
+            <ul class="connect-media">
+                <li><a  href="#"><img src="images/twitter.svg" alt="twitter"></a></li>
+                <li><a  href="#"><img src="images/Linkedinicon.svg" alt="linkedin"></a></li>
+                <li><a  href="#"><img src="images/medium.svg" alt="medium"></a></li>
+                <li><a  href="#"><img src="images/github.svg" alt="github"></a></li>
+                <li><a  href="#"><img src="images/angellist.svg" alt="angellist"></a></li>
+            </ul>
+            <button class="see-more resume">Get my resume</button>
+        </div>
+        <ul class=" about-list">
+            <li class="listItem"><img src="images/enabled.png" alt="Enabled Arrow">Languages</li>
+            <li> 
+                <ul class="about-list languages">
+                    <li> <img src="images/lang3.png" alt="JavaScript image">&nbsp; <h5>JavaScript</h5> </li>
+                    <li><img src="images/lang2.png" alt="HTML image">&nbsp; <h5>HTML</h5> </li>
+                    <li><img src="images/lang1.png" alt="css image">&nbsp; <h5>CSS</h5> </li>
+                </ul>
+            </li>
+            <li class="listItem"><img src="images/Disabled.png" alt="Disabled Arrow">Frameworks <hr> </li>
+            <li class="listItem"><img src="images/Disabled.png" alt="Disabled Arrow">Skills <hr> </li>
+        </ul>
+    </section>
+    <!--Footer section-->
+    <footer class="contact_section">
+        <div class="footer_content">
+            <h2 class="contact-header">Contact me</h2>
+            <p class="contact-message">
+                If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+            </p>
+            <form class="form-container" action="#" method="POST">
+                <input id="name" type="text" maxlength="30" name="full-name" placeholder="Petro Lesapiti" required class="input">
+                <input id="email" type="email" name="email-address" placeholder="petrolesapiti@gmail.com" required class="input">
+                <textarea id="textarea" name="comment" maxlength="500" cols="30" rows="10" placeholder="Write your message here" required class="input text-area"></textarea>
+                <button class="see-more submit" type="submit">Get In Touch</button>
+            </form>
```

Comment:
- When clicked on the button, the form details should be sent to formspree website
- The button interaction for hover, active, and disabled should be styled in css


![contact-1](https://user-images.githubusercontent.com/49184579/177561137-9f5bf7d7-ce83-4dc0-b700-ff0de657a73c.png)

Compare your design and the figma design:
1. The background is grey. Please match the figma design
2. The background image seems to be working, but the blue background is the issue. I see you are using an image for the blue background and hence it is not responsive.
3. The margin/padding is not matching the figma design. Please follow the figma design as closely as possible.